src/mkdocstrings_handlers/python/handler.py

```diff
@@ -130,16 +129,16 @@ class PythonHandler(BaseHandler):
         line_length (int): Maximum line length when formatting code/signatures. Default: `60`.
         merge_init_into_class (bool): Whether to merge the `__init__` method into the class' signature and docstring. Default: `False`.
         show_if_no_docstring (bool): Show the object heading even if it has no docstring or children with docstrings. Default: `False`.
+        show_docstring_description (bool): Whether to display the textual block (including admonitions) in the object's docstring. Default: `True`.
         show_docstring_section_attributes (bool): Whether to display the "Attributes" section in the object's docstring. Default: `True`.
-        show_docstring_section_parameters (bool): Whether to display the "Parameters" section in the object's docstring. Default: `True`.
+        show_docstring_section_examples (bool): Whether to display the "Examples" section in the object's docstring. Default: `True`.
         show_docstring_section_other_parameters (bool): Whether to display the "Other Parameters" section in the object's docstring. Default: `True`.
+        show_docstring_section_parameters (bool): Whether to display the "Parameters" section in the object's docstring. Default: `True`.
         show_docstring_section_raises (bool): Whether to display the "Raises" section in the object's docstring. Default: `True`.
-        show_docstring_section_warns (bool): Whether to display the "Warns" section in the object's docstring. Default: `True`.
-        show_docstring_section_yields (bool): Whether to display the "Yields" section in the object's docstring. Default: `True`.
         show_docstring_section_receives (bool): Whether to display the "Receives" section in the object's docstring. Default: `True`.
         show_docstring_section_returns (bool): Whether to display the "Returns" section in the object's docstring. Default: `True`.
-        show_docstring_section_examples (bool): Whether to display the "Examples" section in the object's docstring. Default: `True`.
-        show_docstring_description (bool): Whether to display the textual block (including admonitions) in the object's docstring. Default: `True`.
+        show_docstring_section_warns (bool): Whether to display the "Warns" section in the object's docstring. Default: `True`.
+        show_docstring_section_yields (bool): Whether to display the "Yields" section in the object's docstring. Default: `True`.
```

Comment:
Forgot to remove the `section_` part :smile: ?

Reply:
oopsies, fixed. sorry bout that

Reply:
No worries!